src/VerifyWindowsHelloController.js

```diff
@@ -95,7 +99,7 @@ function (Okta, FormController, FormType, webauthn, Spinner, FooterSignout, Wind
           'request': '_startEnrollment',
           'error': '_stopEnrollment',
           'abort': '_stopEnrollment',
-          'sync': '_successEnrollment'
```

Comment:
The spinner becomes visible on the `request` event that is at the beginning of the `save` function.
To hide the button bar I have to add the `hide` class to the `'.o-form-button-bar'` and after `render` it becomes visible again.
Unfortunately, If I use the `sync` event my callback is not the last one.
The last one is in the BaseForm.js:
```
this.listenTo(this.model, 'sync', function () {
        if (this.model.get('__edit__')) {
          this.model.set('__edit__', false, {silent: true});
        }
        this.__saveModelState(this.model);
        this.render();
      });
```
and it renders the view again and makes the button visible.

To make it work I trigger the `signIn` event after the `sync` and the button bar stays hidden.